src/gui/editors/PianoRoll.cpp

```diff
@@ -1056,9 +1056,11 @@ void PianoRoll::keyPressEvent(QKeyEvent* ke )
 	{
 		const int key_num = PianoView::getKeyFromKeyEvent( ke ) + ( DefaultOctave - 1 ) * KeysPerOctave;
 
-		if(! ke->isAutoRepeat() && key_num > -1)
+		if (!ke->isAutoRepeat() && key_num > -1)
 		{
-			m_pattern->instrumentTrack()->pianoModel()->handleKeyPress( key_num );
+			m_pattern->instrumentTrack()->pianoModel()->handleKeyPress(key_num);
```

Comment:
Would it make sense to always handle the base note key press/release in the chord functions aswell? That way, the code would get a bit smaller. Though I'm also OK with this solution.

Reply:
I think it would force an additional condition each time a note is played (that is, before playing the chord\note, check if there is a chord - then do what's necessary). In my opinion this is less readable.